builder/amazon/common/step_modify_ami_attributes.go

```diff
@@ -153,7 +155,16 @@ func (s *StepModifyAMIAttributes) Run(ctx context.Context, state multistep.State
 		for name, input := range options {
 			ui.Message(fmt.Sprintf("Modifying: %s", name))
 			input.ImageId = &ami
-			_, err := regionConn.ModifyImageAttribute(input)
+
+			var _ *ec2.ModifyImageAttributeOutput
+			err = retry.Config{
+				Tries:      11,
+				RetryDelay: (&retry.Backoff{InitialBackoff: 200 * time.Millisecond, MaxBackoff: 30 * time.Second, Multiplier: 2}).Linear,
+			}.Run(ctx, func(ctx context.Context) error {
```

Comment:
This will retry 11 times even if there's something wrong with the config, is there a possibility to be more specific here ? 
It could also be that we expect that config to always be correct but then I'd add a comment there explaining this.